### ISSUE TITLE
Panic accessing the stats

### ISSUE THREAD

**Issue by robvanmieghem** [rivine/rivine#26]
```
2017/02/16 10:38:03 http: panic serving 127.0.0.1:62430: runtime error: index out of range
goroutine 59 [running]:
net/http.(*conn).serve.func1(0xc4200a3b00)
	/usr/local/Cellar/go/1.7.3/libexec/src/net/http/server.go:1491 +0x12a
panic(0x49fde0, 0xc42000c120)
	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/panic.go:458 +0x243
github.com/rivine/rivine/modules/wallet.(*Wallet).BlockStakeStats(0xc420190480, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/modules/wallet/transactions.go:122 +0x3c3
github.com/rivine/rivine/api.(*API).walletBlockStakeStats(0xc420071140, 0x785c20, 0xc420218410, 0xc4200ee960, 0x0, 0x0, 0x0)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/api/wallet.go:142 +0x1ba
github.com/rivine/rivine/api.(*API).(github.com/rivine/rivine/api.walletBlockStakeStats)-fm(0x785c20, 0xc420218410, 0xc4200ee960, 0x0, 0x0, 0x0)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/api/api.go:182 +0x66
github.com/rivine/rivine/vendor/github.com/julienschmidt/httprouter.(*Router).ServeHTTP(0xc420204780, 0x785c20, 0xc420218410, 0xc4200ee960)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/vendor/github.com/julienschmidt/httprouter/router.go:344 +0x8a1
github.com/rivine/rivine/api.RequireUserAgent.func1(0x785c20, 0xc420218410, 0xc4200ee960)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/api/api.go:97 +0xba
```

Version: 1df1bbd362f409a4ce5bd90ac40a403ce08b6d1b

This was on a locked wallet and no explorer module.

**Comment by robvanmieghem** [comment]
Different error on the same call:
```
2017/02/16 11:01:49 http: panic serving 127.0.0.1:60725: division by zero
goroutine 62 [running]:
net/http.(*conn).serve.func1(0xc420260900)
	/usr/local/Cellar/go/1.7.3/libexec/src/net/http/server.go:1491 +0x12a
panic(0x472d40, 0xc4204b3360)
	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/panic.go:458 +0x243
math/big.nat.div(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4204be480, 0x1, 0x6, 0x0, ...)
	/usr/local/Cellar/go/1.7.3/libexec/src/math/big/nat.go:525 +0x488
math/big.(*Int).QuoRem(0xc420053870, 0xc420053830, 0xc420053850, 0xc4200537d0, 0xc4204be480, 0x1)
	/usr/local/Cellar/go/1.7.3/libexec/src/math/big/int.go:227 +0xbf
math/big.(*Int).Div(0xc420053870, 0xc420053830, 0xc420053850, 0x5)
	/usr/local/Cellar/go/1.7.3/libexec/src/math/big/int.go:238 +0x74
github.com/rivine/rivine/types.Currency.Div(0xc420170600, 0xc4204be480, 0x1, 0x6, 0x300, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/types/currency.go:83 +0x7d
github.com/rivine/rivine/api.(*API).walletBlockStakeStats(0xc42006f4a0, 0x785c20, 0xc4202295f0, 0xc4200ceff0, 0x0, 0x0, 0x0)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/api/wallet.go:152 +0x548
github.com/rivine/rivine/api.(*API).(github.com/rivine/rivine/api.walletBlockStakeStats)-fm(0x785c20, 0xc4202295f0, 0xc4200ceff0, 0x0, 0x0, 0x0)
	/Users/rob/workspace/go/src/github.com/rivine/rivine/api/api.go:182 +0x66
```

This was with an unlocked wallet and with the explorer module running.

**Comment by nvinken** [comment]
The second error : div by zero is fixed in  commit : c61b1485f88ef912a33676b609753ff812bcf562
The first error is the same issue as in issue #25
